index.html

```diff
@@ -3,11 +3,119 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="/styles.css" />
   </head>
   <body>
     <header>
       <h1>Quiz App</h1>
     </header>
+    <main>
+      <section class="content">
+        <article class="card">
+          <img
+            class="bookmark"
+            src="/assets/bookmark.svg"
+            alt="bookmark"
+            onclick=""
```

Comment:
The `onclick` attribute is not needed here. First of all, we should not develop for the future. It's best to implement it only once you really need it. However, we will not include our JavaScript in our HTML code, we will write it in a separate script file.